segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaConnectionListener.java

```diff
@@ -280,5 +286,8 @@ public void close() {
         if (tlsCertFileModificationMonitor != null) {
             tlsCertFileModificationMonitor.stopMonitoring();
         }
+        if (tokenExpiryHandlerExecutor != null) {
+            tokenExpiryHandlerExecutor.shutdownNow();
```

Comment:
This is the wrong way to shut down an executor. See `ExecutorServiceHelpers.shutdown`. But anyway, see my comment above.

Reply:
Fixed.